autopilot-daemon/pkg/utils/functions.go

```diff
@@ -145,3 +147,32 @@ func CreateJob(healthcheck string) {
 		klog.Info("Couldn't create Job ", err.Error())
 	}
 }
+
+func Initmodels() {
```

Comment:
Move to `utils/prometheus.go` rename to `InitModels` 

Actually, what "models" are we initializing? Should it be `InitHardwareMetrics` or similar? I'll let you make the call